elastic_transport/_transport.py

```diff
@@ -345,26 +345,32 @@ def perform_request(  # type: ignore[return]
                         method,
                         node.base_url,
                         target,
-                        meta.status,
+                        resp.meta.status,
                         time.time() - start_time,
                     )
                 )
 
                 if method != "HEAD":
-                    data = self.serializers.loads(raw_data, meta.mimetype)
+                    body = self.serializers.loads(resp.body, resp.meta.mimetype)
                 else:
-                    data = None
+                    body = None
 
-                if meta.status in retry_on_status:
+                if resp.meta.status in retry_on_status:
                     retry = True
                     # Keep track of the last response we see so we can return
                     # it in case the retried request returns with a transport error.
-                    last_response = TransportApiResponse(meta, data)
+                    last_response = TransportApiResponse(resp.meta, body)
 
             except TransportError as e:
                 _logger.info(
                     "%s %s%s [status:%s duration:%.3fs]"
-                    % (method, node.base_url, target, "N/A", time.time() - start_time)
+                    % (
+                        method,
+                        node.base_url,
```

Comment:
This is a more general topics, we should remove potential sensitive information in the logger. In this case an URL can contains `username:password` and I'm not sure about `target`. We should think to have a filter or a redact strategy, [here](https://relaxdiego.com/2014/07/logging-in-python.html#redacting-logs-using-a-filter) an example.